projects/azure/acs-engine/azure-purge/build-artifact.yaml

```diff
@@ -0,0 +1,27 @@
+kind: BuildConfig
+apiVersion: v1
+metadata:
+  name: azure-purge-artifact
```

Comment:
Our convention re chained builds is to have the build that builds binaries named as $tool-binaries and the build that bind-mounts the binaries in the final image named as $tool. In our case it should be `azure-purge-binaries` and `azure-purge`.